File: TEO/Mav.UI/Mav.Ui/src/app/models/invigilator.listing.model.ts
import { UserAssessmentListingModel } from './search/user.assessment.listing.model';

export class InvigilatorListingModel {

  public static default = new InvigilatorListingModel();

  public id: string;

  public firstName: string;

  public surname: string;

  public email: string;

  public enabled: boolean;

  public latestAssessment?: UserAssessmentListingModel;

}

File: TEO/Mav.Models/SearchModels/InvigilatorListingModel.cs
namespace Mav.Models.SearchModels
{
    public class InvigilatorListingModel
    {
        public string Id { get; set; }

        public string FirstName { get; set; }

        public string Surname { get; set; }

        public string Email { get; set; }
        
        public bool Enabled { get; set; }

        public UserAssessmentListingModel LatestAssessment { get; set; }
    }
}
